nautobot_golden_config/management/commands/run_generate.py

```diff
@@ -0,0 +1,22 @@
+"""Add the run_generate command to nautobot-server."""
```

Comment:
To me this is not a very obvious command name. Would `run_generate_config` be clearer?